source/adios2/toolkit/derived/Function.cpp

```diff
@@ -112,7 +98,7 @@ DerivedData AddFunc(std::vector<DerivedData> inputData, DataType type)
                                                 [](T a, T b) { return a + b; });                   \
         return DerivedData({(void *)addValues, inputData[0].Start, inputData[0].Count});           \
     }
-    ADIOS2_FOREACH_PRIMITIVE_STDTYPE_1ARG(declare_type_add)
```

Comment:
Not that this is wrong, but curious about why the change from PRIMITIVE_STDTYPE to PRIMITIVE_ATTRIBUTE_STDTYPE.   This just drops the complex types, from being function candidates.  Is there some problem with allowing them?

Reply:
It is temporary until we decide how to allocate memory for the derived variables. Right now, I didn't want to deal with complex types when using malloc and memset. I added it as a separate commit so I can revert only this one